cmake/pkg/postinst

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -e
+
+USER_NAME=clio
+GROUP_NAME=clio
+CLIO_PREFIX="/opt/clio"
+CLIO_BIN="$CLIO_PREFIX/bin/clio_server"
+CLIO_CONFIG="$CLIO_PREFIX/etc/config.json"
+
+case "$1" in
+    configure)
+        if ! id -u "$USER_NAME" >/dev/null 2>&1; then
+            adduser --system --quiet \
+                --home /nonexistent --no-create-home \
+                --disabled-password \
+                --group "$GROUP_NAME"
```

Comment:
1. Is this a common thing to add a user when installing a package?
2. Should we remove it on uninstall?